docs/CODE_QUALITY.md

```diff
@@ -10,9 +10,9 @@ The systemd project has a number of code quality tools set up in the source
 tree and on the github infrastructure. Here's an incomprehensive list of the
 available functionality:
 
-1. Use `ninja -C build test` to run the unit tests. Some tests are skipped if
+1. Use `meson test -C build` to run the unit tests. Some tests are skipped if
    no privileges are available, hence consider also running them with `sudo
-   ninja -C build test`. A couple of unit tests are considered "unsafe" (as
+   meson test -C build`. A couple of unit tests are considered "unsafe" (as
    they change system state); to run those too, build with `meson
```

Comment:
Hmm, maybe say `meson setup -Dtests=unsafe` here. Also it might be worth mentioning `meson configure -C build -Dtests=unsafe` to tell people how to tweak an already built dir. It feels inconsistent to use the verb-less form here while a verb-ful form is shown above.